playbooks/tags.json

```diff
@@ -1,6 +1,3 @@
 [
-    "Manual Trigger",
-    "Refrenced",
-    "Scenario",
-    "System Configuration Recommendation"
+    "FortiSOAR Health Assessment"
```

Comment:
should we also add system tags for these PBs?

Reply:
will take in next PR